packages/instant-apps-components/src/components/instant-apps-app-guide/instant-apps-app-guide.scss

```diff
@@ -41,13 +46,41 @@
     flex-grow: 0;
   }
 
-  calcite-button {
-    position: absolute;
-    bottom: 5em;
-    left: 0;
-  }
-
   calcite-icon {
     --calcite-ui-icon-color: var(--calcite-color-brand);
   }
+
+  .esri-content-list {
+    counter-reset: item-counter;
```

Comment:
oooo what's this?

Reply:
Changing this to `counter-set` but basically it sets up a named counter that you can increment across items that match a given selector, so using it here to created the styled numbering for the list items.
Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_counter_styles/Using_CSS_counters#difference_between_counter-set_and_counter-reset